documentation/docs/contribute.md

```diff
@@ -9,35 +9,36 @@ keywords:
 - Identity Initiative
 - Documentation
 - Discord
+- reference
 ---
 
-**Thanks for thinking about contributing to the project! We have the following ways that you can contribute.**
+**Thanks for thinking about contributing to the project! You can contribute using the following methods.**
```

Comment:
I think ways worked better here,  "methods" makes me think of something a bit more complex.

Reply:
Combining them👌